test/tools/jsontests/StateTests.cpp

```diff
@@ -142,17 +142,30 @@ class GeneralTestFixture
 public:
 	GeneralTestFixture()
 	{
-		test::StateTestSuite suite;
-		string casename = boost::unit_test::framework::current_test_case().p_name;
-		if (casename == "stQuadraticComplexityTest" && !test::Options::get().all)
-		{
-			std::cout << "Skipping " << casename << " because --all option is not specified.\n";
-			return;
-		}
-		suite.runAllTestsInFolder(casename);
-	}
+        test::StateTestSuite suite;
+        string casename = boost::unit_test::framework::current_test_case().p_name;
+        boost::filesystem::path suiteFillerPath = suite.getFullPathFiller(casename).parent_path();
+
+        // Check specific test cases
+        static vector<string> timeConsumingTestSuites;
```

Comment:
this should work without `push_back`s
```suggestion
        static vector<string> timeConsumingTestSuites{string{"stStaticCall"}, string{"stSStoreTest"}, string{"stQuadraticComplexityTest"}};
```

Reply:
Or maybe without `string{}`, not sure maybe Windows build would complain without

Reply:
It does